BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/Router/Block/BlockRouter.js

```diff
@@ -33,16 +33,28 @@ class BlockRouter extends AbstractBlockRouter
     getBreadcrumb() {
         return [
             {
-                label: Translator.trans('open_orchestra_backoffice.navigation.contribution.title')
+                label: Translator.trans('open_orchestra_backoffice.menu.contribution.title')
             },
             {
-                label: Translator.trans('open_orchestra_backoffice.navigation.contribution.node'),
+                label: Translator.trans('open_orchestra_backoffice.menu.contribution.node'),
                 link: '#'+Backbone.history.generateUrl('showNodes')
             }
         ]
     }
 
     /**
+     * @inheritdoc
+     */
+    getMenuHighlight() {
+        return {
+            newBlockListComponent : 'navigation-node',
```

Comment:
in all router, the menu highlight is the same for all route. so maybe return only a string